scripts/docker/replace-placeholder.sh

```
@@ -0,0 +1,36 @@
+# The MIT License (MIT)
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)